app/components/app_activity_log_component.rb

```diff
@@ -6,21 +6,52 @@ def initialize(patient_session)
   end
 
   def events_by_day
-    consent_events.group_by { _1[:time].to_fs(:nhsuk_date) }
+    all_events
+      .sort_by { -_1[:time].to_i }
+      .group_by { _1[:time].to_fs(:nhsuk_date) }
+  end
+
+  def all_events
+    [vaccination_events, triage_events, consent_events, session_events].flatten
+  end
+
+  def vaccination_events
+    @patient_session.vaccination_records.map do
+      {
+        title: "Vaccinated with #{_1.vaccine.brand} (#{_1.vaccine.type})",
```

Comment:
We should be using the I18n version of the vaccine type here.

```suggestion
        title: "Vaccinated with #{_1.vaccine.brand} (#{t("vaccines." + _1.vaccine.type)})",
```

Interpolating it like this gets a bit ugly, I've created a helper for the vaccines index and show page that might be re-usable here.

https://github.com/nhsuk/manage-vaccinations-in-schools/pull/1364/files#diff-7357383d3f6e7f3684590630a1692c540ef8769ebee862a97e63b6adef736e49R2

Reply:
Also, this'll break with the current factories, as discussed.

Reply:
There's not much left on that PR, btw, I think it just needs a test and a little tidy up

Reply:
Right, will update once your PR is in 👍

Reply:
Updated!